doc/stdenv/stdenv.chapter.md

```diff
@@ -837,6 +837,10 @@ The distribution phase is intended to produce a source distribution of the packa
 
 #### Variables controlling the distribution phase {#variables-controlling-the-distribution-phase}
 
+##### `doDist` {#var-stdenv-doDist}
+
+Set to true to enable the distribution phase.
+
```

Comment:
What's the default value? Is there a corresponding "dont*"  variable?

Reply:
The default value is "the doDist phase is skipped if the variable is not set". Therefore, there is no dont* variable. For details, see here: https://github.com/NixOS/nixpkgs/blob/9f4143e964bff2d96d2e2c370182b7973a226e07/pkgs/stdenv/generic/setup.sh#L1376

Reply:
Add this as explanation: "By default, `distPhase` is skipped when `doDist` is not set".

Reply:
This seems a bit redundant, as it directly precedes the line documenting the behavior https://github.com/NixOS/nixpkgs/blob/9a1d4ccd2ddf7c5cc7d6719c391e9dc3d3d0b28e/doc/stdenv/stdenv.chapter.md?plain=1#L834-L842

Reply:
OK then. It is a bit strange, tho.